pgmuvi/lightcurve.py

```diff
@@ -328,6 +328,41 @@ def __init__(self, xdata, ydata, yerr=None,
         self.__FITTED_MAP = False
         self.__FITTED_MCMC = False
 
+    @classmethod
+    def from_table(cls, VOTable, **kwargs):
```

Comment:
I find the method name a bit misleading. It doesn't match what `to_table()` does, which is return a Table instance - so by that logic, this should take a Table as input, rather than a file. I think we should aim to do both.

Reply:
Fixed.